vllm/worker/model_runner.py

```diff
@@ -154,10 +155,9 @@ def get_max_block_per_batch(self) -> int:
     def _prepare_prompt(
         self,
         seq_group_metadata_list: List[SequenceGroupMetadata],
-    ) -> Tuple[torch.Tensor, torch.Tensor, AttentionMetadata, List[int],
-               List[int], List[int], List[int], Set[LoRARequest],
-               torch.Tensor]:
-        assert len(seq_group_metadata_list) > 0
+    ) -> Tuple[List[int], List[int], Optional[AttentionMetadataPerStage],
+               List[int], List[int], List[int], List[int], Set[LoRARequest],
+               Optional[torch.Tensor], List[int]]:
```

Comment:
ouch... can we do named tuple or data class here. i know this not your code but this really hurts...